main.c

```diff
@@ -3,35 +3,48 @@
 #include <stdlib.h>
 #include <string.h>
 
+int rowMask[MAX_SIZE], colMask[MAX_SIZE], boxMask[MAX_SIZE];
```

Comment:
### remove , it will show error
PS C:\Users\shash\OneDrive\Documents\GitHub\Sudoku> cd "c:\Users\shash\OneDrive\Documents\GitHub\Sudoku\" ; if ($?) { gcc main.c sudoku.c -o main } ; if ($?) { .\main }
C:/SHASHAN-GCC/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\Users\shash\AppData\Local\Temp\ccWJcIDf.o:sudoku.c:(.bss+0x0): multiple definition of `rowMask'; C:\Users\shash\AppData\Local\Temp\cc2Poyrb.o:main.c:(.bss+0x0): first defined here
C:/SHASHAN-GCC/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\Users\shash\AppData\Local\Temp\ccWJcIDf.o:sudoku.c:(.bss+0x80): multiple definition of `colMask'; C:\Users\shash\AppData\Local\Temp\cc2Poyrb.o:main.c:(.bss+0x80): first defined here
C:/SHASHAN-GCC/bin/../lib/gcc/x86_64-w64-mingw32/10.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:\Users\shash\AppData\Local\Temp\ccWJcIDf.o:sudoku.c:(.bss+0x100): multiple definition of `boxMask'; C:\Users\shash\AppData\Local\Temp\cc2Poyrb.o:main.c:(.bss+0x100): first defined here
collect2.exe: error: ld returned 1 exit status